src/lsp/cobol_ptree/env_division.ml

```diff
@@ -25,12 +25,20 @@ type environment_division =
 [@@deriving ord]
 
 (* ------------- ENVIRONMENT DIVISION / CONFIGURATION SECTION -------------- *)
+(* and configuration_section =
+  {
+    source_computer_paragraph: source_computer_paragraph with_loc option order;
+    object_computer_paragraph: object_computer_paragraph with_loc option order;
+    special_names_paragraph: special_names_paragraph with_loc option order;
+    repository_paragraph: repository_paragraph with_loc option order;   (* +COB2002 *)
+  } *)
 and configuration_section =
   {
     source_computer_paragraph: source_computer_paragraph with_loc option;
     object_computer_paragraph: object_computer_paragraph with_loc option;
     special_names_paragraph: special_names_paragraph with_loc option;
     repository_paragraph: repository_paragraph with_loc option;   (* +COB2002 *)
+    order: permutation_quartet';
```

Comment:
We should not have a prime here (that'd be confusing). At worse the prime is for temporary stuff used during construction of the parse-tree.

Reply:
Yes, the prime is there because I had 2 type that where very similar, but from what I can see you prefer the current prime definition of permutation (recursively defined) instead of the fully defined version (without prime)
So I will remove all the unused stuff in the next commit